### DISCUSSION TITLE
Refund request

### DISCUSSION THREAD

**Discussion by matschcar** [aclap-dev/video-downloadhelper#1424] [License & Purchase requests]
### What's your issue?

Hello! I request refund for my license, since it is not working as it should for downloading from a website I was thinking it was. Thank you for your soon reply!
Mateus

### The 10 first digits (only 10) of your license number or email address

ec7b7fda5e

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by matschcar** [comment]
Actually, it is an issue on Brave (the browser where I purchased the license), and it is working on Edge.

So is it possible you convert my license to Edge and send me the new key?

**Comment by jcfrog** [comment]
Hi,
Sorry we can't swap license.
You have been refunded. Should be available in a few business days.
